tests/accessor/accessor_default_values.h

```diff
@@ -0,0 +1,170 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This file provides functions for tests on accessor default values.
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+#define SYCL_CTS_ACCESSOR_DEFAULT_VALUES_H
+
+#include "accessor_common.h"
+
+#include <type_traits>
+
+namespace accessor_default_values_test {
+
+namespace detail {
+
+constexpr int expected_dims = 1;
+constexpr auto expected_target = sycl::target::device;
+constexpr auto expected_placeholder = sycl::access::placeholder::false_t;
+template <typename T>
+constexpr auto expected_mode =
+    std::is_const_v<T> ? sycl::access_mode::read : sycl::access_mode::read_write
+
+}  // namespace detail
+
+/**
+ * @brief Provides functor that lets verify that local_accessor's template
+ *        parameters have expected default values
+ * @tparam T Current data type
+ * @param type_name Current data type string representation
+ */
+template <typename T>
+class test_for_local_acc {
+ public:
+  void operator()(const std::string& type_name) {
+    auto section_name =
+        "Verify default value for dimensions template parameter for " +
+        type_name + " data type. [local_accessor]";
+    SECTION(section_name) {
+      REQUIRE(std::is_same_v<sycl::local_accessor<T>,
+                             sycl::local_accessor<T, detail::expected_dims>>);
+    }
+  }
+};
+
+/**
+ * @brief Provides functor that lets verify that host_accessor's template
+ *        parameters have expected default values
+ * @tparam T Current data type
+ * @param type_name Current data type string representation
+ */
+
+template <typename T, typename DimensionTI>
```

Comment:
```suggestion
template <typename T, typename DimensionT>
```

Reply:
This comment has been applied in this commit: fdc3712565b19f017f6c64f03c15c0edfeaf25e0